README.md

```diff
@@ -60,50 +61,43 @@ In both options the end result should be a new entry in the `Plugins` array in t
  ]
 }
 ```
+>Note: To ensure proper activation of the plugin, make sure the correct plugin is selected or the desired changes are made in the `.uproject` file.
 
 ## Manually invoking the plugin
 
-The plugin is intended to be used by Visual Studio, so it does not add any UI, commands, or logs to the Unreal Editor.
-It still possible to test it's execution, but running the following command:
-
-The command bellow will run the plugin for the specified project and save the Unreal Engine blueprints information in the output file.
-The optional params help to run the command faster.
+The plugin is designed to be used with Visual Studio, and as such, it does not provide any user interfaces, commands, or logs within the Unreal Editor. However, it is still possible to test the plugin's execution by running the command below: 
```

Comment:
We should make sure the command below is an example, since it uses a made-up `EmptyProject` game. That's the reason why it had a hard-coded engine path as well: it was supposed to be a guideline instead of an exact command to copy and use.